saleor/plugins/webhook/tasks.py

```diff
@@ -158,96 +185,172 @@ def send_webhook_using_google_cloud_pubsub(
+    ] = {
+        WebhookSchemes.HTTP: (send_webhook_using_http, (RequestException,)),
+        WebhookSchemes.HTTPS: (send_webhook_using_http, (RequestException,)),
+        WebhookSchemes.AWS_SQS: (send_webhook_using_aws_sqs, (ClientError,)),
         WebhookSchemes.GOOGLE_CLOUD_PUBSUB: (
             send_webhook_using_google_cloud_pubsub,
-            pubsub_v1.publisher.exceptions.MessageTooLargeError,
+            (pubsub_v1.publisher.exceptions.MessageTooLargeError, RuntimeError),
         ),
     }
-
-    if methods := scheme_matrix.get(parts.scheme.lower()):
-        send_method, send_exception = methods
+    if method := scheme_matrix.get(parts.scheme.lower()):
+        send_method, send_exception = method
         try:
-            with webhooks_opentracing_trace(event_type, domain, app_name=app_name):
-                send_method(target_url, message, domain, signature, event_type)
+            return send_method(
+                target_url,
+                message,
+                domain,
+                signature,
+                event_type,
+            )
         except send_exception as e:
-            task_logger.info("[Webhook] Failed request to %r: %r.", target_url, e)
+            return WebhookResponse(content=str(e), status=EventDeliveryStatus.FAILED)
+    raise ValueError("Unknown webhook scheme: %r" % (parts.scheme,))
+
+
+@app.task(
+    bind=True,
+    retry_backoff=10,
+    retry_kwargs={"max_retries": 5},
+)
+def send_webhook_request(self, event_delivery_id):
+    try:
+        delivery = EventDelivery.objects.select_related("payload", "webhook__app").get(
+            id=event_delivery_id
+        )
+    except EventDelivery.DoesNotExist:
+        logger.error("Event delivery id: %r not found", event_delivery_id)
+        return
+    data = delivery.payload.payload
+    webhook = delivery.webhook
+    domain = Site.objects.get_current().domain
+    attempt = create_attempt(delivery, self.request.id)
+    delivery_status = EventDeliveryStatus.SUCCESS
+    try:
+        with webhooks_opentracing_trace(
+            delivery.event_type, domain, app_name=webhook.app.name
+        ):
+            response = send_webhook_using_scheme_method(
+                webhook.target_url,
+                domain,
+                webhook.secret_key,
+                delivery.event_type,
+                data,
+            )
+        attempt_update(attempt, response)
+        if response.status == EventDeliveryStatus.FAILED:
+            task_logger.info(
+                "[Webhook] Failed request to %r: %r.",
+                webhook.target_url,
+                response.content,
+            )
             try:
                 countdown = self.retry_backoff * (2 ** self.request.retries)
                 self.retry(countdown=countdown, **self.retry_kwargs)
             except MaxRetriesExceededError:
                 task_logger.warning(
                     "[Webhook] Failed request to %r: exceeded retry limit.",
```

Comment:
Can we add here more details related to wehbook?  Something similiar as is added in line 280-283
ID, event types

Reply:
added